jupyter_collaboration/utils.py

```diff
@@ -7,6 +7,10 @@
 
 JUPYTER_COLLABORATION_EVENTS_URI = "https://schema.jupyter.org/jupyter_collaboration/session/v1"
 EVENTS_SCHEMA_PATH = pathlib.Path(__file__).parent / "events" / "session.yaml"
+JUPYTER_COLLABORATION_AWARENESS_EVENTS_URI = (
+    "https://schema.jupyter.org/jupyter_collaboration/awareness/v1"
+)
+AWARENESS_EVENTS_SCHEMA_PATH = pathlib.Path(__file__).parent / "events" / "awareness.yaml"
```

Comment:
I see `pathlib.Path(__file__).parent` is used twice, maybe worth storing it in a variable and not repeat the code?
Also, we can do `from pathlib import Path`.

Reply:
will address